React/tic-tac-toe/src/utils/localStorage.js

```diff
@@ -0,0 +1,20 @@
+export const loadState = () => {
+  try {
+    const serializedState = sessionStorage.getItem('state');
+    if (serializedState === null) {
+      return undefined;
+    }
+    return JSON.parse(serializedState);
+  } catch (err) {
+    return undefined;
```

Comment:
> are you using this? i mean, are you managing the error in some level?

not now, but I added it because Dan did it, just in case I need that in the future